README.md

```diff
@@ -4,6 +4,8 @@ Sourcerer hashes your git repos into intelligent engineering profiles.
 Install/uninstall
 =================
 
+Sourcerer requires JRE to be installed on the system. Install last version from [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+
```

Comment:
JRE should be enough if you only want to run the JAR file that you downloaded. If you want to build the JAR and run it from source code then you have to install JDK.
If you do have troubles with running the JAR file without JDK (only with JRE) we should definitely investigate it.